Zilon.Core/Zilon.Bot.Players.DevelopmentTests/Startup.cs

```diff
@@ -1,31 +1,27 @@
-﻿using LightInject;
+﻿using System;
 
-using Zilon.Bot.Players.LightInject;
-using Zilon.Bot.Players.LightInject.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection;
+
+using Zilon.Bot.Players.NetCore;
+using Zilon.Bot.Players.NetCore.DependencyInjectionExtensions;
 using Zilon.Bot.Players.Strategies;
-using Zilon.Bot.Sdk;
-using Zilon.Core.Tactics.Behaviour;
 using Zilon.Emulation.Common;
-using Zilon.IoC;
 
 namespace Zilon.Bot.Players.DevelopmentTests
 {
-    class Startup: InitialzationBase
+    class Startup : InitialzationBase
     {
-        public override void ConfigureAux(IServiceFactory serviceFactory)
+        public override void ConfigureAux(IServiceProvider serviceFactory)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5028977)